code/modules/martial_arts/combos/judo/goldenblast.dm

```diff
@@ -0,0 +1,20 @@
+/datum/martial_combo/judo/goldenblast
+	//harm disarm harm grab swap disarm disarm grab swap harm disarm disarm grab swap harm
```

Comment:
Comment doesn't match the actual combo?

Reply:
forgot to remove that! that was for me when I was porting it from my notes to code.